Spec/RealtimeClient.swift

```diff
@@ -361,6 +361,8 @@ class RealtimeClient: QuickSpec {
                 if let secs = endInterval {
                     expect(secs).to(beLessThanOrEqualTo(UInt(options.suspendedRetryTimeout)))
                 }
+
+                expect(client.connectionStateTtl).to(equal(120))
```

Comment:
Is there not a way to express this interval `120` with units?

Reply:
Yes, done ab79140